packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -236,7 +246,7 @@ export default class Editor implements IEditor {
                 );
             });
         } else {
-            return queryElements(this.core.contentDiv, selector, callback, scope, null);
+            return queryElements(this.core.contentDiv, selector, callback, scope, undefined);

```

Comment:
add an inline comment to "undefined". e.g.

undefined /*some comment, normally the parameter name */